internal/uvm/create_lcow.go

```diff
@@ -64,6 +64,18 @@ type OptionsLCOW struct {
 	PreferredRootFSType   PreferredRootFSType // If `KernelFile` is `InitrdFile` use `PreferredRootFSTypeInitRd`. If `KernelFile` is `VhdFile` use `PreferredRootFSTypeVHD`
 }
 
+// DefaultLCOWOSBootFilesPath returns the default path used to locate the LCOW
+// OS kernel and root FS files. This default is the subdirectory
+// `LinuxBootFiles` in the directory of the executable that started the current
+// process; or, if it does not exist, `%ProgramFiles%\Linux Containers`.
+func DefaultLCOWOSBootFilesPath() string {
```

Comment:
@jterry75 @jhowardmsft : I will change this if you want. I just wanted to show my thinking on how we can tackle this problem.

- I used the subdirectory name "LinuxBootFiles" because we shouldn't use "LCOW", but I still wanted to keep them organized into a separate directory.
- I searched local paths before global paths. I think this is a better pattern in case e.g. the user wants to have several versions around, and only override the global path for some of them.

Let me know what you think.